src/demo/apps/DialerApp.tsx

```diff
@@ -46,13 +49,16 @@ export function DialerApp(props: AppProps): React.ReactElement {
 
     // Setup listeners
     React.useEffect(() => {
-        const dialListener = fdc3.addIntentListener(fdc3.Intents.DIAL_CALL, (context: Context) => {
-            if (!inCall) {
-                handleIntent(context as ContactContext, false);
-            } else if (context.id && context.id.phone) {
-                setPendingCall(context as ContactContext);
-            }
-        });
+        let dialListener: any;
+        setTimeout(() => {
```

Comment:
Why the timeout?

Reply:
To simulate an app that is slower to start up and set up the listener.  That's the case where the added "listener ready" message is useful.